plugin/parser/differ/mysql/column_test.go

```diff
@@ -185,6 +185,63 @@ func TestColumnComment(t *testing.T) {
 	}
 }
 
+func TestColumnDefaultValue(t *testing.T) {
+	tests := []struct {
+		old  string
+		new  string
+		want string
+	}{
+		{
+			old:  `CREATE TABLE book(name VARCHAR(50) DEFAULT 'Harry Potter' NOT NULL);`,
+			new:  `CREATE TABLE book(name VARCHAR(50) NOT NULL);`,
+			want: "ALTER TABLE `book` MODIFY COLUMN `name` VARCHAR(50) NOT NULL;\n",
+		},
+		{
+			old:  `CREATE TABLE book(name VARCHAR(50) NOT NULL);`,
+			new:  `CREATE TABLE book(name VARCHAR(50) DEFAULT 'Harry Potter' NOT NULL);`,
+			want: "ALTER TABLE `book` MODIFY COLUMN `name` VARCHAR(50) DEFAULT _UTF8MB4'Harry Potter' NOT NULL;\n",
```

Comment:
How about using the RestoreStringWithoutCharset flag when restoring?

Reply:
Improve it in #2751